src/components/BlogForm/BlogForm.jsx

```diff
@@ -122,4 +148,25 @@ const BlogForm = () => {
   );
 };
 
+BlogForm.propTypes = {
+  initialData: PropTypes.shape({
+    id: PropTypes.string,
+    title: PropTypes.string.isRequired,
+    body: PropTypes.string.isRequired,
+    image: PropTypes.string,
+    tags: PropTypes.arrayOf(PropTypes.string).isRequired,
+  }),
+  onSubmit: PropTypes.func,
+};
+
+BlogForm.defaultProps = {
+  initialData: {
+    id: null,
+    title: "",
+    body: "",
+    image: null,
+    tags: [],
+  },
```

Comment:
do not add defaultProps if they are required, you will get warnings in the console.

Reply:
done